recipes/psycopg2/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export LDFLAGS="${LDFLAGS} -L$PREFIX/lib -lssl"
+
+$PYTHON setup.py install
+
+if [[ `uname` == 'Darwin' ]]; then
+     PG_LIB=$(pg_config --libdir)
+     for LIBRARY in `find ${SP_DIR}/${PKG_NAME} -name "*.so"`;
```

Comment:
Just checked the `Travis-CI` log and that is correct.

https://travis-ci.org/conda-forge/staged-recipes/jobs/149016531#L867